test/integration/developers_test.rb

```diff
@@ -66,12 +66,12 @@ class DevelopersTest < ActionDispatch::IntegrationTest
   test "paginating filtered developers respects the filters" do
     with_pagy_default_items(1) do
       get developers_path(sort: :availability)
-      assert_select "h2", count: 1
+      assert_select "h2", count: 2
```

Comment:
Let's make this test more obvious by querying for an `h2` in the desktop form and an `h2` in the mobile form.